bettermdptools/envs/pendulum_discretized.py

```diff
@@ -0,0 +1,316 @@
+"""
+Author: Aleksandr Spiridonov
+BSD 3-Clause License
+"""
+import numpy as np
+from bettermdptools.envs.binning import generate_bin_edges
```

Comment:
@bit-wrangler @jlm429 

I am getting an error on line 6 because there is no `bettermdptools.envs.binning` module

In the future PRs should likely have some sort of unit test or verification assistance to some degree - currently this is broken